scripts/generate_changelog.js

```diff
@@ -49,36 +52,96 @@ if (!fromTag) throw Error("No process.argv[2]");
 if (!toTag) throw Error("No process.argv[3]");
 if (!outpath) throw Error("No process.argv[4]");
 
+/**
+ * @type {Record<string, {heading: string; commitsByScope: Record<string, string[]>}>}
+ */
+const sections = {
```

Comment:
Looking for feedback on what sections, headings and ordering we want to use here.